### PR TITLE
Remove the dependency between create quota command and generators

### PR DESCRIPTION
**Pull request by SaiHarshaK**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR removes the generators dependency from `kubectl create quota`.

**Which issue(s) this PR fixes**:
Part of [#93100](https://github.com/kubernetes/kubernetes/issues/93100)
**Special notes for your reviewer**:
/cc @soltysh

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go [hunk 1]
```diff
@@ -17,12 +17,21 @@ limitations under the License.
 package create
 
 import (
-	"github.com/spf13/cobra"
+	"context"
+	"fmt"
+	"strings"
 
+	"github.com/spf13/cobra"
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go:24]: Nit: split 3-rd party imports and k8s-related imports.

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go [hunk 2]
```diff
@@ -41,14 +50,40 @@ var (
...
+	DryRunVerifier *resourcecli.DryRunVerifier
+
+	genericclioptions.IOStreams
+}
+
+// NewQuotaOpts creates a new *QuotaOpts with sane defaults
+func NewQuotaOpts(ioStreams genericclioptions.IOStreams) *QuotaOpts {
+	return &QuotaOpts{
+		Hard:       "",
+		Scopes:     "",
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go:78]: Nit: empty string is the default value for a string :wink:

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go (hunk 1 above)
Comment (SaiHarshaK) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go:24]: Will do!

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go (hunk 2 above)
Comment (SaiHarshaK) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_quota.go]: Will do!

**Comment by SaiHarshaK** [comment]
@soltysh after updating the nits, the LGTM label got removed. Could you put it again?

**Review by zhouya0 — APPROVED** [review]
/lgtm

**Pull request merged**